### ISSUE TITLE
Experimental results consultation

### ISSUE THREAD

**Issue by codewanghust** [jspunda/prostatex#64]
Hi,
    I'm working on ProstateX-2017 as a machine learning test and without a good performance.So,i want to  use your code as the reference.Because i don't have the label of test set , I would like to consult with you about how your network's performance is ?

**Comment by henkjanhuisman** [comment]
ProstateX-2017 isn't life. Test set not available

**Comment by codewanghust** [comment]
thanks for your reply and what your experiment result is?  e.g.  accuracy and AUC on train set and validation set.

**Comment by henkjanhuisman** [comment]
it is unclear to me what you want

**Comment by codewanghust** [comment]
what I want to know is the best validation accuracy and AUC score on your network.
